src/pe/header.rs

```diff
@@ -4,7 +4,10 @@ use core::iter::FusedIterator;
 use scroll::{IOread, IOwrite, Pread, Pwrite, SizeWith, ctx};
 
 use crate::error;
-use crate::pe::{data_directories, debug, optional_header, section_table, symbol};
+use crate::error::Permissive;
+#[cfg(feature = "te")]
+use crate::pe::data_directories;
+use crate::pe::{debug, optional_header, section_table, symbol};
```

Comment:
Is this change really required? Maybe it should suffice just:

```suggestion
use crate::error::Permissive;
use crate::pe::{data_directories, debug, optional_header, section_table, symbol};
```

Reply:
Ah, okay, so for those TE-specific datas and to avoid warn on unused import without TE feature flag. Makes sense but ig you can remove it from the patch.

https://github.com/m4b/goblin/blob/090a1d437e8c5ab35af6fa0996f5d811230b7264/src/pe/header.rs#L1194-L1197

https://github.com/m4b/goblin/blob/090a1d437e8c5ab35af6fa0996f5d811230b7264/src/pe/header.rs#L1215-L1219